### ISSUE TITLE
Create Welcome Screen for the application

### ISSUE THREAD

**Issue by destrex271** [MicrosoftStudentChapter/notebook#2]

**Comment by devkishor8007** [comment]
@destrex271 if there is any UI design of welcome screen, please provide me - I will do this task!

**Comment by destrex271** [comment]
> @destrex271 if there is any UI design of welcome screen, please provide me - I will do this task!

Check the readme for the link to the figma design. The PR's would be merged on a First Come First Serve basis, so you can start right away no need to wait for any assignment approvals from the maintainers.

**Comment by devkishor8007** [comment]
On the figma, there is blank screen of Welcome screen, should I make the bank or you are making any design?
Just assigned to me, I am doing!

**Comment by destrex271** [comment]
Yup the designs will be finished soon. We are trying to provide chances to contribute to as many people as possible, so you can pick up some other issue till then and revert back to this one if you wish to.

**Comment by devkishor8007** [comment]
glad to know!!
